tests/terraform/graph/checks/resources/CustomAwsEMRSecurityConfiguration/main.tf

```diff
@@ -0,0 +1,91 @@
+provider "aws" {

```

Comment:
Add a test of edge cases where the attribute is `"true"` (every boolean attribute), and "`5`" (you can use data retention attributes which accepts a number)

Reply:
done